indigo/distribution.yaml

```diff
@@ -232,6 +232,22 @@ repositories:
       url: https://github.com/rosalfred/alfred_sr_linux.git
       version: master
     status: developed
+  alliance:
+    release:
+      packages:
+      - alliance
+      - alliance_msgs
+      - rqt_alliance
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/adrianohrl/alliance-release.git
+      version: 1.0.5-0
+    source:
+      test_pull_requests: true
```

Comment:
Have you followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, either remove this line or provide the necessary access as described in the wiki page.

Reply:
It is awaiting ros-pull-request-builder's response.